scrapers/SARJ-LLC.py

```diff
@@ -89,16 +147,35 @@ def scrape_gallery(baseUrl, date, name):
     return map_media(data, studio, baseUrl)
 
 def map_model(baseUrl, model):
+    tags = list(map(lambda t: {'Name'}, model['tags']))
+
+    def add_tag(key, format):
+        nonlocal tags
+        if key in model and model[key] != "":
+            tags.append({
+                'Name': format.format(model[key])
+            })
+
+    add_tag('hair', '{} hair')
+    add_tag('pubicHair', '{} pussy')
+    add_tag('eyes', '{} eyes')
+    add_tag('breasts', '{} breasts')
+
     return {
         'Name': model['name'],
         'Gender': model['gender'].upper(),
         'URL': f"{baseUrl}{model['path']}",
         'Ethnicity': model['ethnicity'],
         'Country': model['country']['name'],
-        'EyeColor': model['eyes'],
+        'HairColor': model['hair'].capitalize(),
+        'hair_color': model['hair'].capitalize(),
+        'EyeColor': model['eyes'].capitalize(),
+        'eye_color': model['eyes'].capitalize(),
```

Comment:
`HairColor` is documented in the scraper configuration, but doesn't work. Found some other scraper which used `hair_color` and does work. Same applies to `EyeColor` / `eye_color` as well.

Reply:
yeah you can have a look at
https://github.com/stashapp/stash/blob/develop/pkg/models/model_scraped_item.go
we need the `json` value to be the same

Reply:
IMO it's a bit strange that these are documented as being in CamelCase like all the others, but being defined in snake_case. But seems like all are actually defined in lowercase to I guess the JSON unmarshaller is case insensitive then.

It just felt a bit odd to use hair_color etc while all others are capitalized. So was also unsure if the desired behavior / name actually shouldn't have been in CamelCase here as well. But seeing the code it seems like more are actually in snake_case.

Reply:
Yeah when used in stash, they have the json definition available so the json marshalller/unmarsaller can use that. I don't know if there is something equivalent in python. Imo for the python part they should all be snake_case but since i have seen a lot of users use the CamelCase i didnt bother to mention that. In some cases though like here we can''t avoid snake_case definitions.